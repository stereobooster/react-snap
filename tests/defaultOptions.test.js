const { defaultOptions } = require("./../index.js");

test("defaultOptions", () => {
  expect(defaultOptions).toMatchSnapshot();
});
