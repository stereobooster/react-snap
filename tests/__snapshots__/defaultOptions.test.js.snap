// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaultOptions 1`] = `
Object {
  "asyncScriptTags": false,
  "cacheAjaxRequests": false,
  "concurrency": 4,
  "crawl": true,
  "destination": null,
  "exclude": Array [],
  "externalServer": false,
  "fixInsertRule": true,
  "fixWebpackChunksIssue": "CRA1",
  "headless": true,
  "http2PushManifest": false,
  "ignoreForPreload": Array [
    "service-worker.js",
  ],
  "include": Array [
    "/",
  ],
  "inlineCss": false,
  "minifyCss": Object {},
  "minifyHtml": Object {
    "collapseBooleanAttributes": true,
    "collapseWhitespace": true,
    "decodeEntities": true,
    "keepClosingSlash": true,
    "sortAttributes": true,
    "sortClassName": false,
  },
  "port": 45678,
  "preconnectThirdParty": true,
  "preloadImages": false,
  "publicPath": "/",
  "puppeteer": Object {
    "cache": true,
  },
  "puppeteerArgs": Array [],
  "puppeteerExecutablePath": undefined,
  "puppeteerIgnoreHTTPSErrors": false,
  "removeBlobs": true,
  "removeScriptTags": false,
  "removeStyleTags": false,
  "saveAs": "html",
  "skipThirdPartyRequests": false,
  "source": "build",
  "sourceMaps": true,
  "userAgent": "ReactSnap",
  "viewport": Object {
    "height": 850,
    "width": 480,
  },
  "waitFor": false,
}
`;
